bundles/ui/org.eclipse.smarthome.ui.classic/snippets/rollerblind.html

```diff
@@ -1,2 +1,2 @@
-<li><img src="images/%icon%.png" width=29 height=29 class="iFull" /><span class="options"><img src="images/_down.png" ontouchstart="buttonPressed('CMD?%item%=DOWN')" onMouseDown="buttonPressed('CMD?%item%=DOWN')" onMouseUp="buttonReleased('CMD?%item%=DOWN', 'CMD?%item%=STOP')" /><img src="images/_stop.png" onclick="ChangeState('CMD?%item%=STOP')" /><img src="images/_up.png" ontouchstart="buttonPressed('CMD?%item%=UP')" onMouseDown="buttonPressed('CMD?%item%=UP')" onMouseUp="buttonReleased('CMD?%item%=UP', 'CMD?%item%=STOP')" /></span style="%labelstyle%">%label%</li>
+<li style="%labelstyle%"><img src="/images/%icon%.png" width="29" height="29" class="iFull" /><span class="options"><img src="images/_down.png" ontouchstart="ESH.buttonPressed('../CMD?%item%=DOWN');event.stopPropagation();event.preventDefault();" ontouchend="ESH.buttonReleased('../CMD?%item%=DOWN', '../CMD?%item%=STOP');event.stopPropagation();event.preventDefault();" onMouseDown="ESH.buttonPressed('../CMD?%item%=DOWN')" onMouseUp="ESH.buttonReleased('../CMD?%item%=DOWN', '../CMD?%item%=STOP')" /><img src="images/_stop.png" onclick="ESH.changeState('../CMD?%item%=STOP')" /><img src="images/_up.png" ontouchstart="ESH.buttonPressed('../CMD?%item%=UP');event.stopPropagation();event.preventDefault();" ontouchend="ESH.buttonReleased('../CMD?%item%=UP', '../CMD?%item%=STOP');event.stopPropagation();event.preventDefault();" onMouseDown="ESH.buttonPressed('../CMD?%item%=UP')" onMouseUp="ESH.buttonReleased('../CMD?%item%=UP', '../CMD?%item%=STOP')" /></span style="%valuestyle%">%label%</li>
```

Comment:
Did you put the absolute path ("/") back on purpose? Please note that these had been removed with https://github.com/eclipse/smarthome/commit/3a5e695fd8fa5b469c17a5f4aba25157f2be61da in order to make it possible to use the UI with reverse proxies on different urls.